readme.md

```diff
@@ -595,6 +595,7 @@
 - [Open Source Maintainers](https://github.com/nayafia/awesome-maintainers#readme) - The experience of being an open source maintainer.
 - [Calculators](https://github.com/xxczaki/awesome-calculators#readme) - Calculators for every platform.
 - [Captcha](https://github.com/ZYSzys/awesome-captcha#readme) - A type of challenge–response test used in computing to determine whether or not the user is human.
+- [Software Design Patterns](https://github.com/DovAmir/awesome-design-patterns) - Software and architecture related design patterns.
```

Comment:
This should explain what "software design patterns" are, not what the list contains.